margin trading

alokym86 opened the pull request.

alokym86 committed:
margin trading

ridicoulous reviewed (changes requested):
mea culpa. In each async method should be passed additional parameter `CancellationToken ct=default`

alokym86 committed:
add the missed parameter to async methods.

rename HitBtc.Net.Objects.MarketData.HitBtcTrade to HitBtcPublicTrade to avoid confusion with HitBtc.Net.Objects.Trading.HitBtcTrade. Added inheriting the second class from first one

ridicoulous reviewed (approved):
thx for contributing
